sync2/poller.go

```diff
@@ -64,6 +64,8 @@ type IPollerMap interface {
 	EnsurePolling(pid PollerID, accessToken, v2since string, isStartup bool, logger zerolog.Logger)
 	NumPollers() int
 	Terminate()
+	MissingTxnID(eventID, userID, deviceID string) (bool, error)
+	SeenTxnID(eventID string) error
```

Comment:
True, but this is just the gadget that decides whether we should send an all-clear; it's the caller's responsibility to publish the payload with txn id. (Although I think even then we don't care what the txn_id is, because of the annotation logic.